src/App.js

```diff
@@ -1,24 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import MainPage from './components/pages/Mainpage';
+import HouseDetails from './components/pages/HouseDetails';
+import ReservePage from './components/pages/ReservePage';
+import DeleteHouse from './components/pages/DeleteHouse';
+import Login from './components/auth/Login';
+import Register from './components/auth/Register';
+// import Navigation from './components/sidebar/Navigation';
+import AddHouse from './components/pages/AddHouse';
+import MyReservation from './components/pages/MyReservation';
+// import './styling/style.css';
+// import './styling/navbar.css';
+// import './styling/mainPage.css';
+import './styling/houseReserve.css';
+import setupAxiosInterceptors from './helpers/setupAxiosInterceptors';
+
+setupAxiosInterceptors();
 
 function App() {
```

Comment:
- [ ] There is no back button for "My reservations" and "AddHouse", both in the desktop and mobile implementations:

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/8c65d4dc-0b14-4aa4-8a40-df4fd380e441)

![image](https://github.com/codeWithHyder/house-rent-frontend/assets/57421823/0606259a-4a4e-493e-a7ec-248778441cfe)

Kindly, add one back button for those pages, so there is a way to go back, just as you did with "Delete House" and "Reserve House".   You could also make your hamburger menu visible on all the mobile pages for this same purpose.

That way, your frontend will be more consistent the UX will be improved. 👍